components/molecule/avatar/src/AvatarFallbackName/index.js

```diff
@@ -21,7 +22,8 @@ const MoleculeAvatarFallbackName = ({
       ? `${firstName.charAt(0)}${lastName.charAt(0)}`
       : firstName.charAt(0)
 
-  const backgroundColor = useConvertStringToHex(nameProp)
+  const stringToHexBackgroundColor = useConvertStringToHex(nameProp)
```

Comment:
backgroundColorProp = stringToHexBackgroundColor || useConvertStringToHex(nameProp)

useConvertStringToHex is a complex fn. It only needs its execution if the color is not proper declared

Reply:
but hooks cannot be called conditionally, I'll change the hook then